package.json

```diff
@@ -1,7 +1,7 @@
 {
   "fitbit": {
     "appType": "clockface",
-    "appDisplayName": "cozie",
+    "appDisplayName": "Cozie-fed-debugging",
```

Comment:
just make sure you don't update the app name and UUID

Reply:
I have changed that.